<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Welcome to FinTrack!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Permissions Portal</string>
    </property>
    <widget class="QPushButton" name="addUsers">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add or Update Users</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Financial Portal</string>
    </property>
    <widget class="QPushButton" name="transactionsButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Transactions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="incomeStatementButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Income Statement</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Accounts Portal</string>
    </property>
    <widget class="QPushButton" name="viewAccounts">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>View All Accounts</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>640</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1470</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
